common/src/main/java/whocraft/tardis_refined/common/block/console/GlobalConsoleBlock.java

```diff
@@ -95,4 +105,65 @@ public void destroy(LevelAccessor levelAccessor, BlockPos blockPos, BlockState b
 
         super.destroy(levelAccessor, blockPos, blockState);
     }
+
+    @Override
+    public void animateTick(BlockState blockState, Level level, BlockPos blockPos, RandomSource randomSource) {
+        super.animateTick(blockState, level, blockPos, randomSource);
+
+        TardisClientData clientData = TardisClientData.getInstance(level.dimension());
+        if (clientData != null) {
+            if (clientData.isFlying() && level.random.nextInt(4) == 0) {
+                var d = (double)blockPos.getX() + randomSource.nextFloat() * 1.25;
```

Comment:
Repetitive particle code. Particle code does not use nice variables. replace d,e,f with xCoord, yCoord and xCoord.